builder/confd/confd.go

```diff
@@ -0,0 +1,95 @@
+// Package confd provides basic Confd support.
+//
+// Right now, this library is highly specific to the needs of the present
+// builder. Because the confd library is not all public, we don't use it directly.
+// Instead, we invoke the CLI.
+package confd
+
+import (
+	"github.com/Masterminds/cookoo"
+	"github.com/Masterminds/cookoo/log"
+	"github.com/Masterminds/cookoo/safely"
+
+	"fmt"
+	"os/exec"
+	"strconv"
+	"time"
+)
```

Comment:
It's idiomatic to specify standard import first and then custom ones. Please see https://github.com/golang/go/wiki/CodeReviewComments#imports